tests/p_iptables/iptables_add-remove_test.sh

```diff
@@ -3,7 +3,7 @@
 
 t_Log "Running $0 - verify iptables can insert and delete rules"
 
-if [ "$centos_ver" = "7" ] || [ "$centos_ver" = "8" ];then
+if [ "$centos_ver" = "7" ] || [ "$centos_ver" = "8" ] || [ "$centos_ver" = "9" ];then
```

Comment:
Can we switch these to `-ge` conditionals? If versions after 7 use firewalld we can then rely on a greater-than test without changing again for CentOS Stream 10

Reply:
Ok, will check for the conditionals, as I said, I mostly just used the style the tests were written in, but you are right -ge makes more sense :)